### PR TITLE
Debian: fix HostonlyInitrd and autologin

### PR DESCRIPTION
**Pull request by bluca**

### CODE REVIEW

File: mkosi/__init__.py
Comment (DaanDeMeyer) [inline mkosi/__init__.py:1435]: The other config files enclose dracut lists in quotes with spaces. If possible, this should do that as well.

File: mkosi/__init__.py
Comment (DaanDeMeyer) [inline mkosi/__init__.py:2890]: Nit, we prefer python f-strings (`f"{device_prefix}pts/0"). Ditto for the other changes.

File: mkosi/__init__.py
Comment (bluca) [reply mkosi/__init__.py]: fixed

File: mkosi/__init__.py
Comment (bluca) [reply mkosi/__init__.py]: fixed

**Review by DaanDeMeyer — APPROVED** [review]
LGTM, let's wait for CI to finish before merging

**Pull request merged**
